README.md

```diff
@@ -73,16 +73,89 @@ for the word "anonymous"
 + Please add other existing providers
-and search for if they allow anonymous feedback ... 
+and search for if they allow anonymous feedback ...
 
 ### Open Source Feedback Systems?
 
 > Are there any _Open Source_ feedback systems we can use and/or learn from.
 
++ ***PHPBack***: http://www.phpback.org/ is an open source project similar to
+Uservoice found at https://github.com/ivandiazwm/phpback/. It's a feedback system
+for products that can be directly injected into your website.
+
+
++ Scarce options for _Open Source_ feedback systems that will fit our exact
+needs. Please add other existing providers and add them to the list.
 
 ### Gather Requirements
 
 > Gather requirements from people you know who either work
 or _have_ worked somewhere they don't _love_.
 What _specifically_ would they like to have in a feedback system?
+
+#### Examples
+
+[@conorc1000](https://github.com/conorc1000):
+##### Previous feedback experience
+Conor worked for a civil engineering company run by a husband and wife. The husband
+was the boss of the company and the wife was the company secretary. They would
+have an appraisal once a year where you could voice your opinion about anything
+regarding the company and your experience within it. However, this appraisal
+was more of a box ticking exercise and the feedback given wouldn't filter
+through to the boss which meant that no action could be taken on it, rendering
+the whole thing a bit pointless.
+##### Ideal feedback experience
+Conor's ideal feedback system would consist of a free text input where you could
+voice your opinion on the organisation that you work with. He would have two
+toggles, one for anonymity and then the other for privacy. By switching the
+toggles on the feedback it would mean that you could attach your name to it, or
+not, and you could also say whether or not you would prefer if the feedback was
+kept private or made public. He said that it might be good to have an incentive
+to give feedback as well so that more people would get involved.
+
+[@roryc89](https://github.com/roryc89)
+##### Previous feedback experience
+Rory worked for a spread-betting platform in London and said that he had similar
+opportunities to give feedback in the form of appraisals. His perception was
+that the feedback wasn't absorbed sufficiently and that it got lost among the
+noise.
+##### Ideal feedback experience
+Rory's ideal feedback experience would be one with the ability to give anonymous
+feedback about anything that was going on within the organisation. Also he would
+want to open discussions about decisions that the agency makes in order to
+further understand the way it functions and why. Rory also believes that
+feedback should have the option to be private so that people are protected if
+neccessary (you might be able to work out the identity of a person just by
+the way they phrase something or the context in which they are talking). 
```

Comment:
Agreed on privacy. ✅ 
There is a branch of "forensic linguistics" that aims to identify people from their writing.
see: https://www.schneier.com/blog/archives/2011/08/identifying_peo_2.html